session/session.go

```diff
@@ -1327,7 +1327,6 @@ func runStmt(ctx context.Context, se *session, s sqlexec.Statement) (rs sqlexec.
 				se.StmtCommit()
 			}
 		}
-		err = finishStmt(ctx, se, err, s)
```

Comment:
Remove this will conflict with https://github.com/pingcap/tidb/pull/19106/files#diff-10dab83c04a99358d431906d37f48b92be97d50a2719d4ddf81c62d3b03a0aebR1228

Maybe we should keep this function call in here and add a flog to avoid duplicate call in line 1339.

Reply:
Forgot it....